scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -111,25 +111,55 @@ trait ClassLikeSupport:
           val target = ExtensionTarget(extSym.symbol.name, extSym.tpt.dokkaType.asSignature, extSym.tpt.symbol.dri)
           parseMethod(dd.symbol, kind = Kind.Extension(target))
         }
-
+      // TODO check given methods?
       case dd: DefDef if !dd.symbol.isHiddenByVisibility && dd.symbol.isGiven =>
-        Some(parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None))) // TODO check given methods?
+        Some(dd.symbol.owner.typeMember(dd.name))
```

Comment:
Can `dd.symbol.owner.typeMember(dd.name)` be a null? If not then simple `if` will be nicer to read:

```
if dd.symbol.owner.typeMember(dd.name).exisits then None
else Some(parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None)))
```